rust/src/disassembly.rs

```diff
@@ -210,3 +216,63 @@ impl Drop for DisassemblyTextLine {
         }
     }
 }
+
+pub use binaryninjacore_sys::BNDisassemblyOption as DisassemblyOption;
```

Comment:
I prefer the type syntax; `pub type DisassemblyOption = BNDisassemblyOption;`

Reply:
Adressed in d06905d844a74aff66667b7afbc262f6da343ba2